src/reactapp/src/components/address/hooks/useCountryState.js

```diff
@@ -0,0 +1,44 @@
+import { useEffect, useMemo } from 'react';
+import _get from 'lodash.get';
+import { useFormikContext } from 'formik';
+
+import useAppContext from '../../../hook/useAppContext';
+import { prepareCountryOptions, prepareCountryStateOptions } from '../utility';
+
+export default function useCountryState({ fields }) {
+  const { values, setFieldValue } = useFormikContext();
+  const [{ countryList, stateList }] = useAppContext();
+  const countrySelected = _get(values, fields.country);
+  const regionField = fields.region;
+  const stateSelected = _get(values, regionField);
+
+  // whenever the country is swiched, we need to clear the state input
```

Comment:
File 52/170 and the first thing I find is a typo in the comment, sorry 😄 

`swiched` => `switched`

Reply:
done 👍

Reply:
done 👍